SpatialGDK/Documentation/content/shooter-tutorial.md

```diff
@@ -0,0 +1,3 @@
+# Get started: 4 - Multi-Server Shooter Tutorial
```

Comment:
lower case "t" for tutorial and we need to standardize on "multi-server" or "mulitserver". We don't have a standard and neither do our standard writing style guides.